search/mongo_search.go

```diff
@@ -14,6 +14,8 @@ import (
 	"gopkg.in/mgo.v2/bson"
 )
 
+var opInterruptedCode = 11601
```

Comment:
This one is tricky - I can't find any resource for MongoDB's internal error codes. The only exit code documentation readily available doesn't list any code over `100`:
https://docs.mongodb.com/manual/reference/exit-codes/